.hound.yml

```diff
@@ -0,0 +1,2 @@
+scss:
+  config_file: .scss-lint.yml
```

Comment:
Actually, it's already disabled https://github.com/it-projects-llc/svalka/blob/master/.hound.yml
But for some reason hould-ci admin option for global config was switched offf